src/lib/db-listeners/db-stars-listener.ts

```diff
@@ -0,0 +1,46 @@
+import { DB } from "../db";
+import { UserStarModel } from "../../models/tools/user-star";
+import { forEach } from "lodash";
+
+export class DBStarsListener {
+  private db: DB;
+  private starsRef: firebase.database.Reference | null = null;
+
+  constructor(db: DB) {
+    this.db = db;
+  }
+
+  public start() {
+    this.starsRef = this.db.firebase.ref(
+      this.db.firebase.getUserDocumentStarsPath(this.db.stores.user)
+    );
+    this.starsRef.on("child_changed", this.handleUpdateStars);
+    this.starsRef.on("child_added", this.handleUpdateStars);
+  }
+
+  public stop() {
+    if (this.starsRef) {
+      this.starsRef.on("child_changed", this.handleUpdateStars);
```

Comment:
```suggestion
      this.starsRef.off("child_changed", this.handleUpdateStars);
```
Note that the same is true in the `db-comments-listener`, so it's clearly a copy/paste error from that PR where it apparently fell through the cracks.

Reply:
Yup, I based `db-stars-listener` on `db-comments-listener`. Surprised we didn't catch this earlier.  I fixed both instances on this branch.